src/makeRobotAccountant.js

```diff
@@ -6,7 +6,15 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfCalls = 0;
+
+  return (firstNumber) => {
+    countOfCalls++;
+
+    return (secondNumber) => (countOfCalls % 2 === 0 && countOfCalls > 3)
+      ? 'Bzzz... Error!'
+      : firstNumber + secondNumber;
```

Comment:
this is not the best way to write code in an arrow function, let's make the code easier to read
```suggestion
    return (secondNumber) => (
      countOfCalls % 2 === 0 && countOfCalls > 3
        ? 'Bzzz... Error!'
        : firstNumber + secondNumber
    )
```